src/test/regress/expected/multi_mx_function_call_delegation.out

```diff
@@ -722,6 +722,13 @@ SET search_path TO multi_mx_function_call_delegation, public;
 select create_distributed_function('mx_call_func(int,int)');
 ERROR:  operation is not allowed on this node
 HINT:  Connect to the coordinator and run it again.
+-- show that we can call delegated function from worker nodes
+SELECT delegated_function(100);
```

Comment:
should we enable debug messages to show it is actually delegated?

Reply:
this is not delegated to another node. We should find another dist key, so try `SELECT delegated_function(101);` etc.
We should see something `DEBUG:  pushing down the procedure` with DEBUG1

I think the appropriate place to add these tests is `multi_mx_function_call_delegation.sql` where we already have tests delegated from the coordinator. We should have tests from workers as well